maps/tether/tether_defines.dm

```diff
@@ -196,34 +201,6 @@
 		Z_LEVEL_PLAINS
 	)
 
-// Short range computers see only the six main levels, others can see the surrounding surface levels.
```

Comment:
How will this change affect surface z-levels of Virgo 3b that are not normally accessible from station, Underdark and Wilderness?

Reply:
They will be accessible to suit sensors, cameras, radio, etc. Unless that's not desirable? I don't see why that'd be a bad thing, though. They already had telecomms because relays were pre-mapped in for all the different levels you mention, so one might think that suit sensors and such would work too.

Reply:
Wilderness is area south of station and it definitely has no relay. My mistake with Underdark, but I do know that its specifically NOT connected in terms of everything else (like suit sensors or GPS). Both Wilderness not having any connection and Underdark having no suitsensors seemed like balance decisions to make sure that risk of being in expedition areas isn't lowered even further than it already is by being on same planet and having weaker mob selection.

Reply:
I would say that this change is probably fine. Especially since someone who does go out there and die will at least have a chance of medical being able to recover the body and such.